### PR TITLE
Final V1 formatting

### PR DESCRIPTION
**Pull request by SpencerMartel**
Final string formatting for V1, consistency should be maintained throughout now.

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ async def handle_add_game_request(message, entry):
         elif status ==  GameAddStatus.FREE_GAME:
             reply = 'This game is free'
         else:
-            reply = (f'**{entry["name"].capitalize()}** was successfully added to our tracking list.\nsteam://openurl/https://store.steampowered.com/app/{entry["appid"]}\n')
+            reply = (f'**{entry["name"].title()}** was successfully added to our tracking list.\nsteam://openurl/https://store.steampowered.com/app/{entry["appid"]}\n')
```
Comment (Nwitz) [inline client.py:91]: I think the game name that comes from Steam is already formatted for example 
"Deep Rock Galactic" or "HITMAN 3" 

Unless entry['name'] doesn't contain that value?

File: client.py [hunk 2]
```diff
@@ -106,11 +106,10 @@ async def handle_delete_game_request(message,entry):
     reply = ''
     if entry != None:
         status = delete_game((entry["appid"], entry["name"]))
-        output = games_were_tracking_string()
         if status == True:
-            reply = f'**{entry["name"]}** was successfully deleted from our tracking list.\n{output}'
+            reply = f'**{entry["name"].title()}** was successfully deleted from our tracking list.'
```
Comment (Nwitz) [inline client.py:102]: Same title concern listed above.

File: client.py [hunk 3]
```diff
@@ -106,11 +106,10 @@ async def handle_delete_game_request(message,entry):
     reply = ''
     if entry != None:
         status = delete_game((entry["appid"], entry["name"]))
-        output = games_were_tracking_string()
         if status == True:
-            reply = f'**{entry["name"]}** was successfully deleted from our tracking list.\n{output}'
+            reply = f'**{entry["name"].title()}** was successfully deleted from our tracking list.'
         else:
-            reply = f'There was a problem deleting the game, are we tracking it?\n{output}'
+            reply = f'There was a problem deleting the game, are we tracking it?'
```
Comment (Nwitz) [inline client.py]: Maybe in this case, it is worth outputting the list so the user can confirm?

File: client.py [hunk 4]
```diff
@@ -127,7 +126,7 @@ def list_games_for_reply():
         if discounted_percent > 0:
             formatted_games += f'\n•\t**{game_title}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
         else:
-            formatted_games += f'\n•\t**{game_title}** is not on sale :smiling_face_with_tear:\n\t  {game_url}'
+            formatted_games += f'\n•\t**{game_title}** is not on sale.\n\t  {game_url}'
```
Comment (Nwitz) [inline client.py]: Should we even say anything here, or just do game title?

File: client.py [hunk 5]
```diff
@@ -149,9 +148,9 @@ async def list_sales(message):
             formatted_discounted_price = discounted_price.replace('CDN$ ','$')
             discounted_percent = games[key]['price_overview']['discount_percent']
             if discounted_percent > 0:
-                formatted_sales += f'\n•\t**{game_title}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
+                formatted_sales += f'\n•\t**{game_title.title()}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
```
Comment (Nwitz) [inline client.py]: Same title concern as mentioned above, games come pre-formatted

File: client.py [hunk 6]
```diff
@@ -149,9 +148,9 @@ async def list_sales(message):
             formatted_discounted_price = discounted_price.replace('CDN$ ','$')
             discounted_percent = games[key]['price_overview']['discount_percent']
             if discounted_percent > 0:
-                formatted_sales += f'\n•\t**{game_title}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
+                formatted_sales += f'\n•\t**{game_title.title()}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
     else:
-        formatted_sales = f'{line_string}\nNo games from your wishlist are currently on sale :sob:'
+        formatted_sales = f'{line_string}\nNo games from your wishlist are currently on sale.'
```
Comment (Nwitz) [inline client.py]: Reasonable place to sob if you wanna keep it in

File: client.py [hunk 7]
```diff
@@ -167,25 +166,15 @@ def games_were_tracking_string():
...
-    await message.reply(f"A day happened {reply}")
-
 def format_game_for_reply(game, game_id):
     url = f'steam://openurl/https://store.steampowered.com/app/{game_id}'
     name = game['name']
     discounted_percent = game['price_overview']['discount_percent']
     discounted_price = game['price_overview']['final_formatted'] 
     formatted_discounted_price = discounted_price.replace('CDN$ ','$')
-    formatted_game = f'**{name}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {url}'
+    formatted_game = f'**{name.title()}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {url}'
```
Comment (Nwitz) [inline client.py]: Same concern for pre-formatted game titles as listed above

File: client.py (hunk 1 above)
Comment (SpencerMartel) [reply client.py:91]: It does but some are all caps and some aren't so I thought it looked neater if we ran it through .title() for cohesion, and functionally we run user_input through .lower() so it doesn't mess with how a user would write their game name

File: client.py (hunk 3 above)
Comment (SpencerMartel) [reply client.py]: it absolutely is

File: client.py (hunk 1 above)
Comment (Nwitz) [reply client.py:91]: Fair enough, but one is for matching and ease of use on the technical side, and the other is a design choice taken by the publisher of the game, and how users will see it in other contexts.

File: client.py (hunk 6 above)
Comment (SpencerMartel) [reply client.py]: would like to sob

File: client.py (hunk 1 above)
Comment (SpencerMartel) [reply client.py:91]: True, I suppose keeping in line with publisher decisions trumps aesthetics. Shall revert.

File: client.py (hunk 6 above)
Comment (Nwitz) [reply client.py]: sob engaged

**Review by Nwitz — APPROVED** [review]
Shweet

**Pull request merged**